apps/condo/domains/ticket/utils/helpers.spec.ts

```diff
@@ -405,6 +407,19 @@ describe('Helpers', () => {
 
         })
 
+        describe('getPageSizeFromQuery', () => {
+            describe('extract page size based on pagesize string from query object if pagesize presnts or set it to default', () => {
+
+                it('is valid when setted to value not from enum', () => {
+                    expect(getPageSizeFromQuery({ pagesize: '16' })).toStrictEqual(20)
+                })
+
+                it('is valid when not defined', () => {
+                    expect(getPageSizeFromQuery({ offset: undefined })).toStrictEqual(TICKET_PAGE_SIZE)
```

Comment:
I don't see a working with `offset` param in `getPageSizeFromQuery`.
Maybe, `pagesize`?

Reply:
Done